From: David Nusinow <dnusinow@debian.org>
Date: Sun, Apr 15 13:11:06 2007 -0400
Forwarded: not-needed
Subject: Change default device permission to 666

Patch libdrm to default to device permission 666 so we don't have to do
it in xorg.conf. The only way libdrm can do anything is through the
server anyway. This can still be overridden by a user's xorg.conf.

--- a/xf86drm.h
+++ b/xf86drm.h
@@ -74,7 +74,7 @@ extern "C" {
 /* Default /dev/dri directory permissions 0755 */
 #define DRM_DEV_DIRMODE	 	\
 	(S_IRUSR|S_IWUSR|S_IXUSR|S_IRGRP|S_IXGRP|S_IROTH|S_IXOTH)
-#define DRM_DEV_MODE	 (S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP)
+#define DRM_DEV_MODE	 (S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP|S_IROTH|S_IWOTH)
 
 #ifdef __OpenBSD__
 #define DRM_DIR_NAME  "/dev"
